### PR TITLE
now generates the value classes and it's project

### PR DESCRIPTION
**Pull request by andersndm**
- Generates value type classes and the value type project

### CODE REVIEW

File: MssBuilder/MssValueTypeBuilder.cs [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+@"namespace {{Namespace}}
+{
+    class {{ClassName}}
+    {
+        public {{TypeName}} {{PropName}} { get; set; }
+
+        public {{ClassName}}() { }
+        public {{ClassName}}({{TypeName}} value) => {{PropName}} = value;
+    }
+}";
```
Comment (edmondbaloku) [inline MssBuilder/MssValueTypeBuilder.cs:24]: Possible to move this to Template folder and use TemplateFactory?

File: MssBuilder/MssValueTypeBuilder.cs [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            {
+                Namespace = ProjectName,
+                ClassName = valueType.Name,
+                TypeName = valueType.Field.Type.ToString(),
+                PropName = valueType.Field.Name
+            };
+            var result = template(data);
+
+            Console.WriteLine(classFile);
+            Console.WriteLine(result);
```
Comment (edmondbaloku) [inline MssBuilder/MssValueTypeBuilder.cs:46]: Does this print the entire class out? If so, should it be removed?

File: MssBuilder/MssValueTypeBuilder.cs [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        {
+            var projectTemplate =
+@"<Project Sdk=""Microsoft.NET.Sdk"">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
```
Comment (edmondbaloku) [inline MssBuilder/MssValueTypeBuilder.cs:61]: Possible to move this to Template as well?

**Review by edmondbaloku — APPROVED** [review]

File: MssBuilder/MssValueTypeBuilder.cs (hunk 3 above)
Comment (andersndm) [reply MssBuilder/MssValueTypeBuilder.cs:61]: moved into templates

File: MssBuilder/MssValueTypeBuilder.cs (hunk 2 above)
Comment (andersndm) [reply MssBuilder/MssValueTypeBuilder.cs:46]: removed

File: MssBuilder/MssValueTypeBuilder.cs (hunk 1 above)
Comment (andersndm) [reply MssBuilder/MssValueTypeBuilder.cs:24]: moved into templates

**Pull request merged**